plugins/filter/accumulate.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Max Gautier <mg@max.gautier.name>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+DOCUMENTATION = '''
+  name: accumulate
+  short_description: Produce a list of accumulated sums of the input list contents.
```

Comment:
`short_description` should not end with a period (it's the only description that should not):
```suggestion
  short_description: Produce a list of accumulated sums of the input list contents
```